src/components/EditMonitorConditions.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+    <div class="monitor-conditions">
+        <label class="form-label">{{ $t("Conditions") }}</label>
+        <div class="monitor-conditions__conditions">
+            <template v-for="(condition, conditionIndex) in model" :key="conditionIndex">
+                <EditMonitorConditionGroup
+                    v-if="condition.type === 'group'"
+                    v-model="model[conditionIndex]"
+                    :is-first="conditionIndex === 0"
+                    :get-new-group="getNewGroup"
+                    :get-new-condition="getNewCondition"
+                    :condition-variables="conditionVariables"
+                    @remove="removeCondition"
+                />
+                <EditMonitorCondition
+                    v-else
+                    v-model="model[conditionIndex]"
+                    :is-first="conditionIndex === 0"
+                    :is-last="conditionIndex === model.length - 1"
+                    :condition-variables="conditionVariables"
+                    @remove="removeCondition"
+                />
+            </template>
+        </div>
+        <div class="monitor-conditions__buttons">
+            <button class="btn btn-outline-secondary btn-sm me-2" type="button" data-testid="add-condition-button" @click="addCondition">
```

Comment:
![image](https://github.com/user-attachments/assets/8b1eba94-29e5-4eed-8a18-525909ae4539)

I have a concern that this will get accidentally tapped by mobile users.

Could space the buttons out more; Or add a confirmation dialog (annoying imo); Or keep it above the fields.

Reply:
I'm experimenting with keeping the variable select aligned with the next rows but it only looks good visually if the next row is a condition. When the next row is a group, it gets whacky. IMO better to just leave it alone.

Reply:
> maybe nicer if aligned

Experimenting with this and it's janky to implement.

Reply:
> X on "Add Condition" button

Not sure what you mean here. Only have Groups at the root level?

Reply:
![image](https://github.com/user-attachments/assets/79302326-2e3e-4aad-aad1-858036a3b327)

Is this necessary? A group is a group. I don't think it needs to be "Add Condition Group".

Reply:
> Not sure what you mean here. Only have Groups at the root level

I was thinking about https://github.com/louislam/uptime-kuma/pull/5048#discussion_r1730428445

=> lets move that subpart there, sorry

Reply:
> Could space the buttons out more;

Lets do that. And maybe make the delete buttons' forground `text-color` red if you think this accidental tapping is a concearn.

Reply:
Updated to:

![image](https://github.com/user-attachments/assets/05fc583a-fef2-451c-a06a-ccbafe45dd7a)